### PR TITLE
LPS-156278 Add GlobalCSSCETsConfiguration tests

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/layout/layout-admin-web/test/js/layout/look_and_feel/GlobalCSSCETsConfiguration.test.tsx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+				]}
+				portletNamespace=""
+				selectGlobalCSSCETsEventName=""
+			/>
+		);
+
+		const addButton = await screen.findByText('add-css-extensions');
+
+		expect(addButton).toBeInstanceOf(HTMLButtonElement);
+		(addButton as HTMLButtonElement).click();
```
[the remark this answers is not in the document]
Comment (p2kmgcl) [reply modules/apps/layout/layout-admin-web/test/js/layout/look_and_feel/GlobalCSSCETsConfiguration.test.tsx:127]: :thinking: you're right, updating that

[the remark this answers is not in the document]
Comment (p2kmgcl) [reply modules/apps/layout/layout-admin-web/test/js/layout/look_and_feel/GlobalCSSCETsConfiguration.test.tsx]: done! I've amended the last commit

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
Nice majo 😍

**Pull request closed without merging**